packages/fxa-auth-server/lib/oauth/logging/index.js

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+let mozlog = null;
+
+module.exports = function(nameOrLog) {
+  if (typeof nameOrLog === 'string') {
+    // ignore logger labels for now.
+    // auth and oauth used different logging strategies
+    // eventually these should be consolidated under
+    // lib/log.js
+    if (mozlog) {
+      return mozlog.logger;
+    }
+    // main key_server must set the log
+    if (process.mainModule.filename.includes('key_server')) {
+      throw new Error('uninitialized mozlog');
+    }
+    // probably a test
+    mozlog = {
+      logger: require('mozlog')({ app: 'unknown', level: 'critical' })(
+        nameOrLog
+      ),
+    };
+    return mozlog.logger;
+  }
+  mozlog = nameOrLog;
+};
```

Comment:
this got shimmed to hell because of how different oauth-server's logging strategy is from auth-server. `/bin/key_server.js` sets the `mozlog` here before any other modules that depend on it get `require`d. gross but works.

Reply:
In a subsequent PR, would it be possible to unify the strategies?

Reply:
That's absolutely the plan